fsharp-backend/tests/Tests/UserDB.Tests.fs

```diff
@@ -1,6 +1,6 @@
 module Tests.UserDB
 
-// Most DB tests can be checked
+// Most DB tests can be checked <- what does this mean by 'checked'?
```

Comment:
pending question

Reply:
I think I probably stopped half-way through the comment. This was supposed to say that most DB tests are in db.tests, and these ones are the ones we couldn't fit into that format.